src/test/java/uk/gov/hmcts/cmc/claimstore/services/FormaliseResponseAcceptanceServiceTest.java

```diff
@@ -384,9 +359,9 @@ public void formaliseSettlementWithCourtDeterminedPaymentIntention() {
             .courtDetermination(CourtDetermination
                 .builder()
                 .courtDecision(paymentIntention)
+                .decisionType(COURT)
```

Comment:
would be nice to have tests for the scenarios when decision type is CLAIMANT/DEFENDANT after determination